integration-tests/funqy-amazon-lambda/src/test/java/io/quarkus/funqy/test/NoArgFunTest.java

```diff
@@ -0,0 +1,39 @@
+package io.quarkus.funqy.test;
+
+import static org.junit.jupiter.api.Assertions.fail;
+
+import java.util.Timer;
+import java.util.TimerTask;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+
+import io.quarkus.amazon.lambda.test.LambdaClient;
+import io.quarkus.test.junit.QuarkusTest;
+
+@QuarkusTest
+@ExtendWith(UseNoArgFunExtension.class)
+public class NoArgFunTest {
+
+    @Test
+    public void testNoArgFun() {
+        Thread mainTestThread = Thread.currentThread();
+        Timer timer = new Timer(true);
+        timer.schedule(new TimerTask() {
+            @Override
+            public void run() {
+                mainTestThread.interrupt();
+            }
+        }, 10_000);
+
+        try {
+            LambdaClient.invoke(String.class, null);
```

Comment:
Maybe we could add overload with timeout?

Reply:
+1